### ISSUE TITLE
PR #438 observation

### ISSUE THREAD

**Issue by acidtib** [basecamp/kamal#448]
hey @djmb great work on the new features, I have a small request Could we have a version bump when big changes like #438 are introduced? this PR just broke so many deployments.

```
Releasing the deploy lock...
  Finished all in 39.4 seconds
  ERROR (SSHKit::Command::Failed): Exception while executing on host xx.xx.xx.xx: docker exit status: 125
docker stdout: Nothing written
docker stderr: docker: open .kamal/env/roles/service_name-web.env: no such file or directory.
See 'docker run --help'.
```

**Comment by djmb** [comment]
Hi and thanks @acidtib!

If you point to the released versions from RubyGems this shouldn't be an issue I think. Are you pulling Kamal directly from the main branch?

**Comment by delivey** [comment]
Same issue happening for me. Have you found a fix?

**Comment by djmb** [comment]
@delivey, how do you install Kamal? If you use `gem install kamal` it should only pull released versions from Rubygems.

**Comment by dhh** [comment]
The new features, with breakage, will be released as part of v1.0. But yeah, if you run off main, you have to expect that things might break!
